packages/hs-react-ui/src/components/TextInput/TextInput.tsx

```diff
@@ -176,24 +154,15 @@ const TextInput = ({
       <InputComponent
         cols={cols}
         rows={rows}
-        aria-label={ariaLabel}
-        placeholder={placeholder}
         onChange={(e: SyntheticEvent) => {
           e.persist();
           onChange(e);
           debouncedChange(e);
         }}
-        onKeyPress={onKeyPress}
-        onKeyDown={onKeyDown}
-        onKeyUp={onKeyUp}
-        onFocus={onFocus}
-        onBlur={onBlur}
-        onReset={onReset}
-        onInput={onInput}
         value={value || defaultValue}
```

Comment:
We shouldn't do this anymore - they'll pass either value or default value and we should respect that and not put both into value

Reply:
Gotcha, I am going to remove them altogether so they are just passed through.